README.md

```diff
@@ -319,13 +319,15 @@ You can contribute by [reporting issues and making feature requests](https://git
 
 ## Documentation
 
-| Link                                                            | Description                                                               |
-|-----------------------------------------------------------------|---------------------------------------------------------------------------|
-| [README_LINUX](README_LINUX.md)                                 | Linux specific setup instructions                                         |
-| [Release Notes](Documentation/ReleaseNotes.md)                  | Release notes and known issues per major release                          |
-| [Gameplay Configuration](Documentation/GamplayConfiguration.md) | How to adjust gameplay settings                                           |
-| [SettingsScreen](Documentation/SettingsScreen.md)               | How to use and extend the settings screen                                 |
-| [Packaging MPS](Documentation/PackedAssetBuilds.md)             | How to build and package MPS for distribution or running servers remotely |
+| Link                                                                             | Description                                                               |
+|----------------------------------------------------------------------------------|---------------------------------------------------------------------------|
+| [README_LINUX](README_LINUX.md)                                                  | Linux specific setup instructions                                         |
+| [Release Notes](Documentation/ReleaseNotes.md)                                   | Release notes and known issues per major release                          |
+| [Gameplay Configuration](Documentation/GamplayConfiguration.md)                  | How to adjust gameplay settings                                           |
+| [SettingsScreen](Documentation/SettingsScreen.md)                                | How to use and extend the settings screen                                 |
+| [Packaging MPS](Documentation/PackedAssetBuilds.md)                              | How to build and package MPS for distribution or running servers remotely |
+| [GameLift Setup](Documentation/Hosting/AmazonGameLift/GameLift.md)               | How to deploy to GameLift                                                 |
```

Comment:
Consider having single doc link here, which in turn can reference the testing document.

Reply:
Fixed. Links to single doc inside the gem (found this is the natural place to look during walk-the-engine). From that single doc are links to setting up gamelift and GL local